scipy/spatial/transform/_rigid_transformation.pyx

```diff
@@ -846,6 +846,20 @@ cdef class RigidTransformation:
         This implements the exponential map that converts 6-dimensional real
         vectors to SE(3).
 
+        An exponential coordinate vector consists of 6 elements - 3 for rotation (known
+        as a rotation vector) and 3 for translation. The exponential mapping can be
```

Comment:
> 3 for translation

From this you could get the impression that the last 3 components directly encode the translation, which is *usually* not the case, since it is multiplied by the inverse Jacobian of SO(3).

Reply:
What can be a better wording? I know it is connected also with screw-motion and differential equations and so on, but I don't want to bring it here.

Reply:
I would just say "3 to encode translation". This doesn't give the impression that they can be directly interpreted as translation.